cypress/e2e/awx/administration/management-jobs.cy.ts

```diff
@@ -18,128 +40,107 @@ describe('Management Jobs', () => {
       cy.wait('@getManagementJobsListPage')
         .its('response.body.results')
         .then((results: SystemJobTemplate[]) => {
-          const responseJobsList = results?.map((job: SystemJobTemplate) => job.name);
-          expect(responseJobsList).to.have.members(managementJobsList);
-          cy.get('.pf-v5-c-table__tbody tr').should('have.length', responseJobsList.length);
-          cy.get('.pf-v5-c-table__tbody tr').each(($row, index) => {
-            cy.wrap($row).find('td').eq(0).should('contain', responseJobsList[index]);
-          });
+          const jobId = results.find((job) => job.name === jobName)?.id;
+          if (jobId) {
+            cy.intercept('POST', `/api/v2/system_job_templates/${jobId}/launch/`).as('launchJob');
+            cy.clickTableRowAction('name', jobName, 'launch-management-job', {
+              inKebab: false,
+              disableFilter: true,
+            });
+            cy.wait('@launchJob')
+              .its('response.body.id')
+              .then((jobId: string) => {
+                cy.verifyPageTitle(jobName);
+                cy.url().should('include', `/jobs/management/${jobId}/output`);
+                cy.waitForManagementJobStatus(jobId).then((response: SystemJobTemplate) => {
+                  const status = response.status;
+                  let resultStatus;
+                  if (status === 'successful') {
+                    resultStatus = 'Success';
+                  }
```

Comment:
resultStatus will be `undefined` if status is any value but `successful`. `resultStatus` should be assigned a value for all status options.

Reply:
yep, already seeing it tests failed. looking